File: src/main/java/nl/crosscode/x509certbuddy/X509CertWrapper.java
package nl.crosscode.x509certbuddy;

import java.security.cert.X509Certificate;

public class X509CertWrapper {
    private final X509Certificate cert;

    public X509CertWrapper(X509Certificate cert) {
        this.cert = cert;
    }

    public X509Certificate getCert() {
        return cert;
    }

    public String toString() {
        return cert.getSubjectDN().getName() + " (0x" + cert.getSerialNumber().toString(16)+")";
    }

}

File: src/main/kotlin/nl/crosscode/x509certbuddy/wrappers/X509CertWrapper.kt
package nl.crosscode.x509certbuddy.wrappers

import java.security.cert.X509Certificate

class X509CertWrapper(val cert: X509Certificate) {
    override fun toString(): String {
        return cert.subjectX500Principal.name + " (0x" + cert.serialNumber.toString(16) + ")"
    }
}
